### ISSUE TITLE
The return of "Authentication failed"

### ISSUE THREAD

**Issue by clienthax** [Godzil/Crunchy#14]
Here we go again..

```
Error: Authentication failed:
    at Request._callback (/usr/local/lib/node_modules/crunchy/dist/request.js:82:29)
    at Request.self.callback (/usr/local/lib/node_modules/crunchy/node_modules/request/request.js:187:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
    at Request.<anonymous> (/usr/local/lib/node_modules/crunchy/node_modules/request/request.js:1044:10)
    at emitOne (events.js:77:13)
    at Request.emit (events.js:169:7)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/crunchy/node_modules/request/request.js:965:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
```

**Comment by clienthax** [comment]
@majewskim any ideas?

**Comment by majewskim** [comment]
@clienthax will have a look today!

**Comment by Godzil** [comment]
That was my fear, the problem is using the RPC was not a good idea, they were going to phase that out.
There are some way to "bypass" the cloudfare check, I will add that and revert the login using the token, the same way as the website use, it is the more secure way. I will try to update that in the week

**Comment by majewskim** [comment]
@Godzil Last time that happened I actually did something similar to what https://github.com/codemanki/cloudscraper does (I didn't know about this repo before :) ).
This should work, maybe it's worth giving it a go. I will have some time today so I can prototype on my fork.

**Comment by Godzil** [comment]
Fixed on my side, I let it finish to check if it's working, then I submit the changes and publish an updated version!

**Comment by Godzil** [comment]
Fixed in version 1.1.15

**Comment by clienthax** [comment]
@Godzil Can you push this to npm?

**Comment by Godzil** [comment]
The login fix has been pushed yesterday (v1.1.15)
